src/ts/background/index.ts

```diff
@@ -32,6 +31,11 @@ const handleSetBadge = async (request: MessageGeneric<any>) => {
   }
 };
 
+const handleSetBadge = async (request: MessageGeneric<any>) => {
+  console.log(`Handling set badge as got: ${JSON.stringify(request)}`);
```

Comment:
`debugLog`? (the entire file is missing the `debugLog` statement; I know you said you haven't updated it everywhere, just pointing this out in case you'd like to update OTG)